kazoo/client.py

```diff
@@ -1644,8 +1656,100 @@ def reconfig_async(self, joining, leaving, new_members, from_config):
 
         return async_result
 
+    def add_watch(self, path, watch, mode):
+        """Add a watch.
+
+        This method adds persistent watches.  Unlike the data and
+        child watches which may be set by calls to
+        :meth:`KazooClient.exists`, :meth:`KazooClient.get`, and
+        :meth:`KazooClient.get_children`, persistent watches are not
+        removed after being triggered.
+
+        To remove a persistent watch, use
+        :meth:`KazooClient.remove_all_watches` with an argument of
+        :attr:`~kazoo.states.WatcherType.ANY`.
```

Comment:
i think all references to `~kazoo.states` should be replaced with `~kazoo.protocol.states`